pos_return_order/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
```

Comment:
You can (and should) put your name, but in only one line, please, or the comment section becomes very big. See https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 for example